CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ option(SSG_OVAL_SCHEMATRON_VALIDATION_ENABLED "If enabled, schematron validation
 option(SSG_SHELLCHECK_BASH_FIXES_VALIDATION_ENABLED "If enabled, shellcheck validation of bash fixes will be performed as part of the ctest tests. Shellcheck tests don't pass right now, this option is discouraged until that's fixed." FALSE)
 option(SSG_LINKCHECKER_VALIDATION_ENABLED "If enabled, linkchecker will be used to validate URLs in all the HTML guides and tables." TRUE)
 option(SSG_SVG_IN_XCCDF_ENABLED "If enabled, the built XCCDFs will include the SVG SCAP Security Guide logo." TRUE)
+option(SSG_JINJA2_CACHE_ENABLED "If enabled, the jinja2 templating files will be cached into bytecode. Also see SSG_JINJA2_CACHE_DIR." TRUE)
+set(SSG_JINJA2_CACHE_DIR "${CMAKE_BINARY_DIR}/jinja2_cache" CACHE PATH "Where should the jinja2 cached bytecode be stored. This speeds up builds at the expense of disk space. You can use one location for multiple SSG builds if necessary.")
```

Comment:
`s/if necessary/for performance improvements/`
`s/Where should the jinja2 cached bytecode be stored/Where the jinja2 cached bytecode should be stored/`

Reply:
Fixed, thx!